packages/client/src/store/selectors/bookings/slots.ts

```diff
@@ -176,3 +178,19 @@ export const getBookingsForCalendar = (
     }, [] as BookingsList)
     .sort((a, b) => (a > b ? -1 : 1));
 };
+
+/**
+ * Get subscribed slots from state for a specific month
+ * @param state Local Redux Store
+ * @returns record of subscribed slots
+ */
+export const getBookedSlotsByMonth =
+  (month: number) =>
+  (state: LocalStore): Record<string, CustomerBookingEntry> =>
+    Object.entries(state.firestore.data?.bookedSlots || {}).reduce(
+      (acc, [slotId, CustomerBookingEntry]) =>
+        DateTime.fromISO(CustomerBookingEntry.date).month === month
+          ? { ...acc, [slotId]: CustomerBookingEntry }
+          : acc,
+      {}
+    );
```

Comment:
None of the changes to this file are necessary if you can apply the update to calendar creation logic I've described above